From 6b4853ed195f0859dbc107b0c4444bbad10b1fe2 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Tue, 21 Mar 2023 11:50:45 +0530
Subject: [PATCH] [Temporary] Commenting the hard coded ndk version

This is commented in the dev branch for ANdroid U
It will be reverted later

Tracked-On: NA
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 groups/device-specific/caas/compatibility_matrix.xml | 4 ++--
 groups/device-specific/caas/framework_manifest.xml   | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/groups/device-specific/caas/compatibility_matrix.xml b/groups/device-specific/caas/compatibility_matrix.xml
index 4389dac..d9af65e 100644
--- a/groups/device-specific/caas/compatibility_matrix.xml
+++ b/groups/device-specific/caas/compatibility_matrix.xml
@@ -47,7 +47,7 @@
             <instance>default</instance>
         </interface>
     </hal>
-    <vendor-ndk>
+    <!--vendor-ndk>
         <version>33</version>
-    </vendor-ndk>
+    </vendor-ndk-->
 </compatibility-matrix>
diff --git a/groups/device-specific/caas/framework_manifest.xml b/groups/device-specific/caas/framework_manifest.xml
index 0ecda8d..45672a4 100644
--- a/groups/device-specific/caas/framework_manifest.xml
+++ b/groups/device-specific/caas/framework_manifest.xml
@@ -200,7 +200,7 @@
             <instance>default</instance>
         </interface>
     </hal>
-    <vendor-ndk>
+    <!--vendor-ndk>
         <version>33</version>
-    </vendor-ndk>
+    </vendor-ndk-->
 </manifest>
-- 
2.17.1

